src/memjs/memjs.ts

```diff
@@ -60,9 +59,18 @@ interface SerializerProp<Value, Extras> {
   serializer: Serializer<Value, Extras>;
 }
 
-type IfBuffer<Value, Extras, IsBuffer, NotBuffer> = Value extends Buffer
+/**
+ * If Value and Extras are of type Buffer, then return type WhenBuffer. Otherwise,
+ * return type NotBuffer.
```

Comment:
mention that this is just for hacky tests?

Reply:
It's not though?